src/arch/c_interface/forte_c.cpp

```diff
@@ -11,42 +11,38 @@
  *  Tarik Terzimehic - make OPC UA server port setable from the command line
  *******************************************************************************/
 
-#include "forte_instance.h"
-#include "forteinstance.h"
-#include <forte_printer.h>
-
-#include "fortenew.h"
-#include "forte_architecture.h"
-#include <stdio.h>
-#include <stdlib.h>
-#include "../../stdfblib/ita/RMT_DEV.h"
+#include "forte_c.h"
 
 #include "../utils/mainparam_utils.h"
+#include "forte_architecture.h"
+#include "forte_printer.h"
+#include "forteinstance.h"
 
-unsigned const int cgForteDefaultPort = 61499;
+static const unsigned int scForteDefaultPort = 61499;
```

Comment:
the coding guidelines doesn't know the prefix sc for static const, only cg for const global.

While file-scope globals theoretically have less impact, they are only an `extern` keyword, used by someone else, away being visible in other compilation units as well.

If we want to code the difference of usage into the name, I would rather suggest cf for "constant file" instead of 
sg "static const", as static const doesn't imply anything than the variable is some sort of global variable and lives in this compilation unit. "cf" would also be more in line with "cg" for const global,

i will mark this only here, to not split up the discussion on several review comments, but my comment is valid for all "sc" changes 

@azoitl @mx990 additional opinions are appreciated

Reply:
I would also recommend to add the "U" suffix to the number, as its an unsigned constant

Reply:
I would strongly suggest to use an [unnamed namespace](https://en.cppreference.com/w/cpp/language/namespace#Unnamed_namespaces) in C++ instead of `static` linkage.

Reply:
I went with the `unnamed namespace` since it's more c++ style. Regarding the naming convetion, since there's no defined convention for constant variables in unnames namespaces I didn't add any prefix to avoid having a "convention" only in that file which is not documented anywhere